apps/web/pages/settings/my-account/out-of-office/index.tsx

```diff
@@ -104,20 +106,17 @@ const OutOfOfficeSection = () => {
               </div>
             )}
           </div>
-          <div className="mt-4 flex flex-row">
+          <div className="mt-4 grid grid-rows-2 gap-2 md:grid-cols-2">
             {profileRedirect && (
-              <div className="mr-2 w-1/2 lg:w-1/3">
+              <div className="">
```

Comment:
Small NIT but I think we can clean up this empty className

Reply:
Remove it, can you re-approve @joeauyeung ?